cmd/process-agent/main_common.go

```diff
@@ -153,6 +151,14 @@ func runApp(ctx context.Context, globalParams *command.GlobalParams) error {
 			return workloadmeta.Params{AgentType: catalog}
 		}),
 
+		// Provide the corresponding tagger Params to configure the tagger
+		fx.Provide(func(c config.Component) tagger.Params {
+			if c.GetBool("process_config.remote_tagger") {
+				return tagger.NewNodeRemoteTaggerParams()
+			}
+			return tagger.NewTaggerParams()
+		}),
```

Comment:
additional comment/idea. it seems that we don't use exactly the same parameter depending on the agent (agent, process-agent) that instantiate the tagger. 
But:
we should be able to know which agent flavour is use (I think we already have a function somewhere)

when we can do a function in the `tagger` package
```golang
// package tagger
// ...
type ParamsFuncType func(c config.Component) Params
func GenerateTaggerParamsFunc(agentFlavour string) ParamsFuncType {
  switch agentFlavour {
  case "agent":
    // ...
  case "process-agent:
    return func(c config.Component) tagger.Params {
      if c.GetBool("process_config.remote_tagger") {
        return tagger.NewNodeRemoteTaggerParams()
      }
	  return tagger.NewTaggerParams()
    }
  }
}